.github/workflows/ci.yml

```diff
@@ -1,218 +1,191 @@
-on:
+---
+'on':
   - pull_request
   - push
-
 name: CI
-
 jobs:
   php-lint:
     name: PHP linter
-
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         php-version:
           - 7.2
           - 7.3
           - 7.4
-
     steps:
-    - name: Checkout
-      uses: actions/checkout@v1
-
-    - name: Install PHP
-      uses: shivammathur/setup-php@v2
-      with:
-        php-version: ${{ matrix.php-version }}
-
-    - name: Run PHP lint
-      run: composer ci:php:lint
-
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '${{ matrix.php-version }}'
+      - name: Run PHP lint
+        run: 'composer ci:php:lint'
```

Comment:
@lukaszuznanski I find this PR quite hard to read due to the mix of formatting changes and other "real" changes. Would you mind creating a PR for the formatting changes that we can merge first? That would be very helpyful for me. Thanks!

Reply:
(For the other files, this is fine as there are only formatting changes as far as I can see. Only this file has the mix of formatting changes and new/changed code.)

Reply:
I get your point. I will separate those changes.

Reply:
Thanks!